Kickstarter-iOS/Features/ManagePledge/Controllers/ManagePledgeViewController.swift

```diff
@@ -538,7 +538,6 @@ final class ManagePledgeViewController: UIViewController, MessageBannerViewContr
   // MARK: - Functions
 
   private func goToRewards(_ project: Project) {
-    /// Render rewards carousel that has the shipping location dropdown
```

Comment:
Removing this comment. It's no longer needed now that the no-shipping flow is the standard.